lib/ConcurrentCompilationError.js

```diff
@@ -7,7 +7,9 @@
 
 const WebpackError = require("./WebpackError");
 
-module.exports = class ConcurrentCompilationError extends WebpackError {
+module.exports = class ConcurrentCompilationError extends (
+	WebpackError
```

Comment:
It is regression bug in `prettier`, I recommend to avoid updating right now

Reply:
Do you have an issue for that?

Reply:
https://github.com/prettier/prettier/issues/9740